### PR TITLE
First attempt from ricardopieper, class CalculateAverage_ricardopieper

### PR DESCRIPTION
**Pull request by ricardopieper**
#### Check List:
- [X] Tests pass (`./test.sh <username>` shows no differences between expected and actual outputs)
- [X] All formatting changes by the build are committed
- [X] Your launch script is named `calculate_average_<username>.sh` (make sure to match casing of your GH user name) and is executable
- [X] Output matches that of `calculate_average_baseline.sh`
* Execution time: 9.7s
* Execution time of reference implementation: 3min19.36s

 My setup is: MacBook Pro M1 2021, 16GB RAM

Notes:
 - Actually my second attempt, the first PR didn't get merged
 - 9.7s on the ./create-measurements3.sh at 1.7GB/s. With the original ./create-measurements.sh it takes 5.5s at 2.2GB/s
 - Yes I know my splitFile is convoluted. Sorry... I wrote it once in the beginning and never looked back.

### CODE REVIEW

File: src/main/java/dev/morling/onebrc/CalculateAverage_ricardopieper.java [hunk 1]
```diff
@@ -0,0 +1,437 @@
+package dev.morling.onebrc;
```
Comment (gunnarmorling) [inline src/main/java/dev/morling/onebrc/CalculateAverage_ricardopieper.java:1]: Please run mvn license:format to add the missing header.

**Review by gunnarmorling — CHANGES_REQUESTED** [review]
One comment inline.

File: src/main/java/dev/morling/onebrc/CalculateAverage_ricardopieper.java (hunk 1 above)
Comment (ricardopieper) [reply src/main/java/dev/morling/onebrc/CalculateAverage_ricardopieper.java:16]: Done!

**Comment by gunnarmorling** [comment]
00:07.925 on the eval machine. Nice!

**Pull request merged**